### ISSUE TITLE
Module not found: Can't resolve 'metro4-react'

### ISSUE THREAD

**Issue by lutfiarfianto** [olton/Metro4-React#14]
my code like this
```
import React from 'react';
import logo from './logo.svg';

import {Grid,Row,Cell,Input} from 'metro4-react' ;

```
then resulting error

```
Failed to compile
./src/App.js
Module not found: Can't resolve 'metro4-react' in 'D:\react_test\m4_react\src'
```
please help me, 
thanks

**Comment by olton** [comment]
Hi!
for 0.1.1 use 
```
import {Grid,Row,Cell,Input} from 'metro4-react\src' ;
```
and you must add rules for webpack module:
```
            {
                test: /\.(js|jsx)$/,
                exclude: /node_modules\/(?!(metro4-react)\/).*/,
                resolve: {
                    extensions: ['.js', '.jsx']
                },
                loader: 'babel-loader',
                options: { babelrcRoots: ['.', '../'] }
            },

            {
                test: /\.(less)$/,
                exclude: /node_modules\/(?!(metro4-react)\/).*/,
                use: [
                    {
                        loader: MiniCssExtractPlugin.loader,
                        options: {
                            hmr: !isProduction,
                            reloadAll: true
                        },
                    },
                    'css-loader',
                    {
                        loader: 'postcss-loader',
                        options: {
                            plugins: [
                                autoprefixer({
                                    overrideBrowserslist :['IE 10', 'IE 11', 'last 3 version']
                                })
                            ],
                            sourceMap: true
                        }
                    },
                    "less-loader"
                ]
            }

```

**Comment by lutfiarfianto** [comment]
thanks, the hack worked

but i had another problem

```
./node_modules/metro4-react/src/components/collapse/collapse.jsx 10:24
Module parse failed: Unexpected token (10:24)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 
| export default class Collapse extends React.Component {
>     static defaultProps = {
|         className: "collapse-css-transition",
|         style: {}

```

maybe we need to add more dependency like babel, webpack etc
i have not try it but think its a good idea
thanks

**Comment by olton** [comment]
https://github.com/olton/Metro4-React/issues/14#issuecomment-544227212

**Comment by olton** [comment]
https://github.com/olton/Metro4-React-App

**Comment by olton** [comment]
static `defaultProps` fixed in 0.1.3
